packages/calcite-components/src/components/modal/modal.scss

```diff
@@ -407,16 +406,4 @@ slot[name="primary"] {
   }
 }
 
-/**
- * Conditional styles for when Modal is slotted in Shell
- */
-
-.container.slotted-in-shell {
```

Comment:
Are these no longer needed? if so, what about the class for slotted-in-shell? We can remove it if not necessary. I think there is a prop as well.

Reply:
I managed to eliminate the prop by dynamically checking if the overflowing container is the document body.